Add more resources (#1205)
MetaMask/metamask-docs: snaps/reference/resources.md

## Primary resources

- [Snaps homepage](https://metamask.io/snaps/)
- [Snaps directory](https://snaps.metamask.io/) - A directory of allowlisted Snaps you can try in the MetaMask extension.
- [SIPs](https://github.com/MetaMask/SIPs) - Suggest new Snaps APIs with Snaps Improvement Proposals.

> - [Builder Team Space](https://go.metamask.io/snaps-builders) - Connect with our builder engagement team and get support while you are building your Snap.

Reviewer: ```suggestion
- [Builder Team Space](https://go.metamask.io/snaps-builders) - Connect with the Snaps builder engagement team and get support while building your Snap.
```

## Example Snaps

- [Example Snaps](https://github.com/MetaMask/snaps/tree/main/packages/examples) - A directory of
  Snaps maintained by MetaMask.

> - [Simple Keyring Snap](https://github.com/MetaMask/snap-simple-keyring) - An example of how to interface with the [Keyring API](../reference/keyring-api/index.md).

Reviewer: ```suggestion
- [Simple Keyring Snap](https://github.com/MetaMask/snap-simple-keyring) - An example account management Snap using the [Keyring API](../reference/keyring-api/index.md).
```

  ([Slides](https://docs.google.com/presentation/d/1ZjhYF-3mwGmsFdcbDgqgFR6t3YIab4_Hk3dRAWjvSQg/edit?usp=sharing))
- [Building the Future with MetaMask Snaps](https://www.youtube.com/watch?v=iE8CGzadKZ8&t=288s) (22 min)
  ([Slides](https://docs.google.com/presentation/d/1LG8MqRrbb9qSg4m8ZjJXPQFccb9YPc-6387hSNpscpY/edit?usp=sharing))

> - [MetaMask Snaps YouTube Playlist](https://www.youtube.com/playlist?list=PLJ8kQp5OiaEM6ad6mC1NmJCGJSZm7cBfI)

Reviewer: ```suggestion
- [MetaMask Snaps YouTube playlist](https://www.youtube.com/playlist?list=PLJ8kQp5OiaEM6ad6mC1NmJCGJSZm7cBfI)
```
